sklearn/model_selection/tests/test_search.py

```diff
@@ -490,16 +490,18 @@ def test_grid_search_sparse():
     assert_equal(C, C2)
 
 
+@pytest.mark.filterwarnings('ignore: The default of the `iid`')  # 0.22
+@pytest.mark.filterwarnings('ignore: The default value of cv')  # 0.22
 def test_grid_search_sparsetarget():
     # Multi-label with sparse target
     X_, y_ = make_multilabel_classification(n_samples=100,
                                             return_indicator=True,
                                             random_state=0)
     y_sparse = sp.csr_matrix(y_)
     clf = MockClassifier(foo_param=0)
-    gs = GridSearchCV(clf, {'foo_param': [0.1, 1.]}, cv=5)
+    gs = GridSearchCV(clf, {'foo_param': [0.1, 1.]})
```

Comment:
What was the motivation of removing `cv=5`?

Reply:
@thomasjpfan I thought that this would not really matter since `cv=5` will be the default setting. Also, in the other tests in `test_search.py`, the `cv` parameter of `GridSearchCV` is usually not set (and a pytest decorator catches the warning associated to the default value of `cv`). If you want, I can keep `cv=5` as it was.